app/service/interfaces/i_login_handler.py

```diff
@@ -0,0 +1,29 @@
+import abc
+from app.utility.base_object import BaseObject
+
+
+class LoginHandlerInterface(abc.ABC, BaseObject):
+    def __init__(self, name):
+        self._name = name
+
+    @abc.abstractmethod
+    async def handle_login(self, request, **kwargs):
+        """Handle login request
+
+        :param request:
+        :return: the response/location of where the user is trying to navigate
+        """
+        pass
+
+    @abc.abstractmethod
+    async def handle_login_redirect(self, request, **kwargs):
+        """Handle redirect to login
+
+        :param request:
+        :return: the response/location of where the user is trying to navigate
```

Comment:
The default handler raises the response rather than returning it. Does this need to be clarified?

Reply:
Yes, I'll clarify accordingly